content/pages/sprint/2021/08end-devenv.rst

```diff
@@ -0,0 +1,75 @@
+============================================
+Scisprint 2021 End of August: solvcon devenv
+============================================
+
+:date: 2021-07-25 14:15
+:url: sprint/2021/08end-devenv
+:save_as: sprint/2021/08end-devenv.html
+
+Scisprint 2021 End of August: solvcon devenv
```

Comment:
Maybe this sprint can host multiple projects, including swportal and Alan's bot.